WordPress/Sniffs/Theme/NoFaviconSniff.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for hardcoded favicons instead of using core implementation.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_NoFaviconSniff implements PHP_CodeSniffer_Sniff {
+
+	const REGEX_TEMPLATE = '` (%s)`i';
+	const REGEX_ATTR_TEMPLATE = '%1$s=[\'"]%2$s[\'"]';
```

Comment:
Replace these two lines like so and I believe the unit tests will pass (fingers crossed - still doing this without running actual tests :sunglasses: ):
```php
const REGEX_TEMPLATE = '` (?:%s)`i';
const REGEX_ATTR_TEMPLATE = '%1$s=[\'"](?:%2$s)[\'"]';
```

Change in the first line isn't really needed, but can't hurt either.